### PR TITLE
Add NXP icon

### LINKED ISSUE #8747: Add NXP Semiconductors Icon
### Brand Name

NXP Semiconductors

### Website

https://www.nxp.com/

### Popularity Metric

The Similarweb rank is 33,570. See https://www.similarweb.com/website/nxp.com/

### Official Resources for Icon and Color

SVG: https://www.ti.com/etc/designs/ti/images/ui/ic-logo.svg
Color: #CC0000
Logo and use guidelines: https://www.nxp.com/company/about-nxp/newsroom/media-library:MEDIA-LIBRARY

### Additional Comments

The prevalent logo is the full color one with overlapping letters, but in the guidelines there is a section called `Logo exception—solid logo` which details how the logo should appear in case it is required to be monochromatic. It would be good to use this version.

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
nxpsemiconductors

**Issue:** closes #8747 

**Similarweb rank:** 33,570. See https://www.similarweb.com/website/nxp.com/

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
~~Hex Value picked from logo based on the most dominant color: `#F9B500`~~
Brand mentions not to use company colors on solid logo. Mentions to only use black or white. Hence color set to `#000000`

Logo extracted from PDF found on brand website that specifically mentions a different logo for monochromatic use: https://www.nxp.com/docs/en/supporting-information/NXPLOGOGD.pdf

```
PS X:\~~~> npm test    

 22284 passing (1s)

PS X:\~~~> npm run lint
--------------------- Summary ---------------------
✓ 2472 valid files.
```

### CODE REVIEW

**Comment by LitoMore** [comment]
image

Please follow their [logo use guideline](https://www.nxp.com/docs/en/supporting-information/NXPLOGOGD.pdf).

**Comment by PratyushJayachandran** [comment]
I had this doubt beforehand.
So should the logo be black or white?

**Comment by LitoMore** [comment]
Black.

**Comment by PratyushJayachandran** [comment]
> Black.

I have changed the color to black and updated PR details to match.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8025,6 +8025,17 @@
...
             "source": "https://nx.dev"
         },
+        {
+            "title": "NXP Semiconductors",
+            "hex": "000000",
+            "aliases": {
+                "aka": [
+                    "NXP"
+                ]
+            },
```
[the remark this answers is not in the document]
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Fixed and pushed commit.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -8025,6 +8025,12 @@
             "hex": "143055",
             "source": "https://nx.dev"
         },
+        {
+            "title": "NXP Semiconductors",
```
Comment (LitoMore) [inline _data/simple-icons.json:8029]:
```suggestion
            "title": "NXP",
```

**Review by LitoMore — CHANGES_REQUESTED** [review]

**Review by LitoMore — APPROVED** [review]

**Pull request merged**